doc/services/pm/device.rst

```diff
@@ -158,8 +158,42 @@ a particular device. It is important to emphasize that, although the
 state is tracked by the subsystem, it is the responsibility of each device driver
 to handle device actions(:c:enum:`pm_device_action`) which change device state.
 
-Each :c:enum:`pm_device_action` have a direct an unambiguous relationship with
-a :c:enum:`pm_device_state`.
+Each :c:enum:`pm_device_state` state is defined as follows:
+
+- ``OFF``: Device hardware is not powered. Once this state is exited, the device
+  is considered to be in its reset state. When a device driver is initialized, the
+  device is presumed to be in this state, it is the initial state. We assume that
+  the device has been left in this state, or the ``SUSPENDED`` state, by the
+  previous "owner" of the device, be that a bootloader, an SoC (reset), or a
+  deinitialized device driver.
+
+  A device driver may be deinitialized when the device is in this state as this
+  is the same state the device driver will be initialized from.
+
+- ``SUSPENDED``: Device hardware is powered, but the device is not needed by the
+  system. The device should be disabled in this state. No device driver API
+  calls will occur in this state. The device will be powered off if its power
+  domain is no longer needed by the system. Note that this state is **NOT** a
+  "low-power"/"partially operable" state, those are configured using device
```

Comment:
I think I understand what you are getting at here , but it should be clarified that while this isn't a low power "operation" state, the device **should** be configured into its lowest power internal state when entering `SUSPENDED`.

Reply:
That makes sense to me